css/src/core/bare-elements.scss

```diff
@@ -32,3 +32,12 @@ fieldset {
 	min-width: 0; // as a replaced element min-width:0 is required to enable it to work in responsive layouts
 	border: none;
 }
+
+::target-text {
+	background-color: $info-background;
+	color: $info-dark;
+}
+
+a > ::target-text {
+	color: $info-dark;
+}
```

Comment:
I'm a little nervous about making highlighted links match the overall text color, especially since we also remove underlines. Can you help me understand the direction for this?

Reply:
@ellynliu you can actually use our standard link color, I just checked and it passes accessibility color contrast requirements with purple background:

```
color: $hyperlink;
```

Reply:
We'll need another color for visited links.

Reply:
Added $hyperlink and $visited as colors for links